doc/configuring.rst

```diff
@@ -185,26 +189,25 @@ Each configuration description is a python dictionary with the following possibl
 ..
 
 name
-    Specifies the name of the configuration. The name is used as the title of the build log
-    block corresponding to the build of this configuration. It is also used to generate
+    Specifies the name of a `build step`_. The name is used as the title of the build log
+    block corresponding to the execution of this step. It is also used to generate
     name of the log file if the option for storing to log files is selected.
-    A project configuration can have no name, but it is not recommended for aesthetic reasons.
-    If several project configurations have the same names, and logs are stored to files
+    A build step can have no name, but it is not recommended for aesthetic reasons.
+    If several build steps have the same names, and logs are stored to files
     (see ``--out`` / ``-o``  `command-line parameter <args.html#Output>`__ for details),
-    all logs for such configurations will be stored to one file in order of their appearances.
+    all logs for such steps will be stored to one file in order of their appearances.
 
 ..
 
 command
-    The list with command line for the configuration launch. For every project configuration
-    first list item should be a console command (e.g. script name, or any other command like ``ls``),
-    and other list items should be the arguments, added to the command (e.g. ``--debug`` or ``-la``).
-    Every command line element, separated with space character, should be passed as a separate
-    string argument. Lists like ``["ls -a"]`` will not be processed correctly and thus
-    should be splat into ``["ls", "-a"]``. Lists like ``["build.sh", "--platform A"]``
-    will not be processed correctly and thus should be plat into ``["build.sh", "--platform", "A"]``.
-    A project configuration can have an empty list as a command. Such configuration
-    won't do anything except informing user about missing it.
+    The list with command line for the build step launch. For every step the first list item should be
+    a console command (e.g. script name, or any other command like ``ls``), and other list items should be
+    the arguments, added to the command (e.g. ``--debug`` or ``-la``).
+    Every command line element, separated with space character, should be passed as a separate string argument.
+    Lists like ``["ls -a"]`` will not be processed correctly and therefore should be splat into ``["ls", "-a"]``.
+    Lists like ``["build.sh", "--platform A"]`` will not be processed correctly and thus should be plat into
+    ``["build.sh", "--platform", "A"]``. A build step can have an empty list as a command.
+    Such step won't do anything except informing the user about having nothing to do.
```

Comment:
I would rephrase it like below:
Such step won't do anything except showing up the step name in Universum execution logs.